Module/MyPythonNode.cs

```diff
@@ -148,9 +191,17 @@ public class MyPythonNode : MyScriptableNode, IMyVariableBranchViewNodeBase
 
         public class Node
         {
+            public struct DashboardProxy
+            {
+                public PythonDictionary set;
```

Comment:
If the struct is needed, then at least rename (set is a keyword, and it's public so at least Set. And it's not a set but a dictionary :-) maybe "Data"?)